modules/dnn/src/layers/region_layer.cpp

```diff
@@ -83,6 +84,7 @@ class RegionLayerImpl CV_FINAL : public RegionLayer
         useLogistic = params.get<bool>("logistic", false);
         nmsThreshold = params.get<float>("nms_threshold", 0.4);
         scale_x_y = params.get<float>("scale_x_y", 1.0); // Yolov4
+        new_coords = params.get<int>("new_coords", 0); // Yolov4x-mish
```

Comment:
Could you please update `supportBackend()` and disable non-OPENCV backends - add `&& new_coords == 0`

Reply:
Done